modules/assigned_identity/assigned_identity.tf

```diff
@@ -1,11 +1,11 @@
 resource "azurerm_user_assigned_identity" "databricks_identity" {
   name                = "databricks-sql-identity"
-  resource_group_name = var.resource_group_name
+  resource_group_name = "ingest00-main-${var.env}"
```

Comment:
Ideally this should be a generic re-usable module, so we shouldn't be hardcoding dlrm-data-ingest names